cmd/boulder-ca/main.go

```diff
@@ -124,6 +181,34 @@ func loadSigner(issuerConfig ca_config.IssuerConfig, cert *x509.Certificate) (cr
 		pkcs11Config.TokenLabel, pkcs11Config.PIN, cert.PublicKey)
 }
 
+func loadBoulderIssuers(profileConfig issuance.ProfileConfig, issuerConfigs []issuance.IssuerConfig, ignoredLints []string) ([]*issuance.Issuer, error) {
```

Comment:
This function is effectively the "heart" of the change -- everything else in this diff is in the service of making this function (and the code which calls it) at clean as it is.